src/sagas/application.js

```diff
@@ -1,6 +1,6 @@
 import { put, select, take, takeEvery } from "redux-saga/effects";
 import { getNetwork, getNetworkPending, types } from "../reducers/application";
-import { setNewWeb3, getWeb3 } from "../services/web3";
+import { getWeb3, setNewWeb3 } from "../services/web3";
```

Comment:
You mentioned we wont need web3 anymore right?

Reply:
will do in another MR but yeah =)

Reply:
https://github.com/TradeTrust/tradetrust-website/pull/12

I think I will create another MR (that this link) to handle it